### PR TITLE
Python edits

### PR DESCRIPTION
**Pull request by martoe3301**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # python-basics
 just python basics but add quality to the code
+
```
Comment (Gaurav258258) [inline README.md]: In my openion it does't affect what you use(Command line/ Terminal). It depends how you use.

File: ex1.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+print("hello world!")
+print("hello world again")
+print("I like typing this.")
+print("this is fun.")
+print('Yah! printing')
+print("I'd much rather you 'not'.")
+print ('I "said" do not touch this.')
```
Comment (Gaurav258258) [inline ex1.py]: print('Python is Best')

File: ex11.1.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+print("How old are you?")
+age = int(input())  #you can use int(input())
```
Comment (Gaurav258258) [inline ex11.1.py]: you can also use eval
+ age=eval(input()) #1 It helps if your age is fractional like 12 year 6 month(12.6)

File: README.md (hunk 1 above)
Comment (martoe3301) [reply README.md]: Thanks for sharing @Gaurav258258

File: ex1.py (hunk 2 above)
Comment (martoe3301) [inline ex1.py]: print('Python is best')

File: ex11.1.py (hunk 3 above)
Comment (martoe3301) [inline ex11.1.py]: age=eval(input())

**Pull request merged**